src/Raven.Server/Documents/DocumentsStorage.cs

```diff
@@ -1207,6 +1208,7 @@ private static Tombstone TableValueToTombstone(JsonOperationContext context, ref
                 result.Collection = TableValueToId(context, (int)TombstoneTable.Collection, ref tvr);
                 result.Flags = TableValueToFlags((int)TombstoneTable.Flags, ref tvr);
                 result.LastModified = TableValueToDateTime((int)TombstoneTable.LastModified, ref tvr);
+                result.LowerId = UnwrapLowerIdIfNeeded(context, result.LowerId);
```

Comment:
`result.LowerId` is `LazyStringValue`. Although here we force to materialize it as `string` because that's the argument of `UnwrapLowerIdIfNeeded`.

Please also see my comment below

Reply:
How this affects all GetTombstonesFrom methods? E.g. What are we expecting to export via Smuggler? Let's talk about this on Sunday/Monday.